Userland/Libraries/LibGUI/TextEditor.cpp

```diff
@@ -856,6 +856,16 @@ void TextEditor::keydown_event(KeyEvent& event)
         return;
     }
 
+    if (event.key() == KeyCode::Key_Tab) {
+        if (has_selection()) {
+            if (event.modifiers() == Mod_Shift) {
+                unindent_selection();
```

Comment:
You can't use a function that only exists in a subsequent commit, it breaks bisecting.